### DISCUSSION TITLE
Is Nextjs right for me?

### DISCUSSION THREAD

**Discussion by chetstone** [vercel/next.js#17924] [Help] [answered]
After working through the awesome Nextjs tutorial (I don't think I've ever seen a better tech tutorial), I'm enamored. But is nextjs overkill for my needs? I need a site for a user manual for my react-native app, and I don't need server side rendering.  

I was initially attracted to nextjs because I thought I might be able to port parts my current site, which have a lot of interactive client-side react code. But after thinking about it more, I'm inclined to put that stuff out of its misery and just start over with a completely static site. Should I be looking at Jekyll instead?

Also, I totally suck at CSS. Are there templates available that can give me a good head start on simple responsive structure for user docs?

**Comment by jamesmosier** [comment, marked as answer]
Hi @chetstone!

You could look at using the [static export functionality](https://nextjs.org/docs/advanced-features/static-html-export) if you wanted a more lightweight approach but still remain in the React/Next ecosystem. 

As far as templates I personally haven't come across any but i'm sure some are out there! 

I'm a big fan of Next, so I always start with it for most any project because of its versatility, but I'm sure other solutions might work out for you although they might not be as flexible if the projects needs ever grow.

**Comment by chetstone** [comment]
Ah! I didn't know about static export. I had gotten the impression from [this article](https://sosnowski.dev/post/static-serverless-site-with-nextjs) that a server was required just for simple routing even if you didn't need SSR. 

This information helps a lot. Thanks!

**Reply by angad777** [reply]
Hey @chetstone 

For some quick and easy to use templates, you can use prebuilt ones like [these from Foundation](https://get.foundation/templates.html)

Throw in the CDN link and you should be good to go.
